beginners/easy/002.positive-negative/206.saadman-galib.positive-negative.py

```diff
@@ -0,0 +1,8 @@
+input_num = int(input("Put your input here: "))
+
+if input_num < 0:
+    print("This is a negative number")
+elif input_num > 0:
+    print("This is a positive number")
```

Comment:
end a statement by using a newline. For now, don't edit it remember this thing

Reply:
Ok bhaiya